resources/assets/components/InboxItem/index.js

```diff
@@ -64,6 +64,7 @@ class InboxItem extends React.Component {
               {campaign['reportback_info']['noun']} {campaign['reportback_info']['verb']}
             </div>
           </article>
+          <a href="#" onClick={e => this.props.showHistory(post['id'], e)}>Edit | Show History</a>
```

Comment:
should these be wrapped in there own `</a>` tags. I feel like the way it is formatted makes it look like they are two distinct links. I forgot if `edit` goes someplace different than `show history`, but if they are in fact the same, maybe we should do "edit/show history" ?